.github/fabricbot.json

```diff
@@ -996,7 +996,7 @@
     "subCapability": "IssueCommentResponder",
     "version": "1.0",
     "config": {
-      "taskName": "Replace `needs more info` label with `needs further triage` label when the author comments on an issue",
+      "taskName": "Replace `needs-author-action` label with `needs further triage` label when the author comments on an issue",
```

Comment:
Do we want to fix the spaces `needs further triage` label as well?

Reply:
I expect this to be addressed comprehensively via #62032. The goal of this PR is to merge two similar labels.